src/applications/claims-status/actions/index.js

```diff
@@ -356,6 +361,70 @@ export function clearAdditionalEvidenceNotification() {
   };
 }
 
+// Helper to build upload success notification message
```

Comment:
The `submitFiles` function was so large that the "cognitive complexity" lint warning was being raised. With that said, I move the notification message code into it's own function.